_posts/2015-04-13-permadelete-on-yank.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Policy change: Yank will now delete the .gem file
+layout: post
+author: Arthur Nogueira Neves
+author_email: arthurnn@gmail.com
+---
+
+The gem yank action, is a command gem authors can call to yank a version out of the rubygems index. There was some expectation that the yank command would also delete the .gem file, however that was not the case, and people would just open a request on help.rubygems.org to actually delete the gem. For a while we only deleted gems manually, using a script, but that was getting out of hand, as the script had to be ran manually.
```

Comment:
I wouldn’t use the word “yank” to define the behavior of `gem yank`. Consider “remove” instead.